ml_genn/ml_genn/compilers/event_prop_compiler.py

```diff
@@ -1410,8 +1457,16 @@ def _build_in_hid_neuron_model(self, pop: Population,
                     "neurons defined in terms of AutoNeuronModel")
             
             # Build adjoint system from model
-            dl_dt, adjoint_jumps, saved_vars_timestep, saved_vars_spike =\
-                self._build_adjoint_system(model, False, self.reg_lambda != 0.0)
+            learn_params= [] if pop not in compile_state.optimisers \
+                else compile_state.optimisers[pop]
```

Comment:
Aren't these dictionaries?
```suggestion
            learn_params = compile_state.optimisers.get(pop, {})
```